cfpb/consumerfinance.gov
# Wagtail: Fix supported_languages variable typo and alphabetize languages in list

## Changes

- Wagtail: Fix supported_languages variable typo and alphabetize languages in list.


## How to test this PR

1. Pull branch, run migrations.
2. Edit or create a page in wagtail and go to the configuration tab and see that the language drop-down is now alphabetized.

## Screenshots

<img width="747" alt="Screen Shot 2021-06-11 at 6 02 33 PM" src="https://user-images.githubusercontent.com/704760/121753044-7e468280-cadf-11eb-943e-9d69f1d06597.png">
